### DISCUSSION TITLE
dead server, resque access

### DISCUSSION THREAD

**Discussion by josok64** [teslamate-org/teslamate#3734] [General]
Is it possible to move all relevant data, like the docker volumes to a new server? Last backup is 3 months old

**Comment by cwanja** [comment]
Feel like I have read about someone doing it before either here or TMC. But do not have a documented process.

**Comment by josok64** [comment]
Old server back to live, so this is not interesting any more. The problem with this Hetzner server was not booting after update. Restoring a backup image, same problem, boot hangs on "Cannot open access to console, the root account is locked" The problem was that 1 year or so ago i added a backup volume to fstab During boot that volume did probably not respond fast enough and booting process stopped and hanged because debian uses sulogin at that stage and the debian installer does not make a root account to login with. Never saw any error about that failed mounting of the (totally functional) backup volume either. Resque mode, chrooting and cleaned fstab from the backup volume entry. Thanks for listening, although this has nothing to do with teslamate of course. Sorry for that!

**Comment by josok64** [comment]
Not teslamate item sorry
